assets/images/dashboard-settings/payment/skrill.svg

```diff
@@ -0,0 +1,22 @@
+</defs>
+</svg>
```

Comment:
All these `svg` files have been used in the pull request #1370 . Update the location with that PR so that same file cannot be added twice. Update your file location by synchronizing with the locations used in this.